WalletWasabi.Fluent/Views/Wallets/Settings/WalletCoinJoinSettingsView.axaml

```diff
@@ -10,46 +10,45 @@
              x:Class="WalletWasabi.Fluent.Views.Wallets.Settings.WalletCoinJoinSettingsView"
              x:CompileBindings="True">
 
-  <StackPanel Spacing="20">
+  <StackPanel Classes="settingsLayout">
 
-    <StackPanel Classes="settingsLayout">
+    <DockPanel>
+      <TextBlock Text="Automatically start coinjoin" />
+      <ToggleSwitch IsChecked="{Binding AutoCoinJoin, Mode=OneWay}" Command="{Binding SetAutoCoinJoin}" />
+    </DockPanel>
 
-      <DockPanel>
-        <TextBlock Text="Automatically start coinjoin" />
-        <ToggleSwitch IsChecked="{Binding AutoCoinJoin, Mode=OneWay}" Command="{Binding SetAutoCoinJoin}" />
-      </DockPanel>
+    <DockPanel>
+      <TextBlock Text="Coinjoin strategy:" VerticalAlignment="Center" DockPanel.Dock="Left" Margin="0" />
+      <Button Theme="{StaticResource AccentButton}" Content="Change" Command="{Binding SelectCoinjoinProfileCommand}" DockPanel.Dock="Right" />
+      <TextBlock Text="{Binding SelectedCoinjoinProfileName}" VerticalAlignment="Center" MinWidth="120" Margin="10 0 10 0" />
+    </DockPanel>
 
-      <StackPanel Spacing="10"
-                  ToolTip.Tip="Coinjoin will not automatically start if the wallet balance is less than this.">
-        <TextBlock Text="Auto-start coinjoin threshold" />
-        <CurrencyEntryBox Classes="standalone" Text="{Binding PlebStopThreshold}" CurrencyCode="BTC" />
-      </StackPanel>
-
-      <DockPanel>
-        <TextBlock Text="Coinjoin strategy:" VerticalAlignment="Center" DockPanel.Dock="Left" Margin="0" />
-        <Button Theme="{StaticResource AccentButton}" Content="Change" Command="{Binding SelectCoinjoinProfileCommand}" DockPanel.Dock="Right" />
-        <TextBlock Text="{Binding SelectedCoinjoinProfileName}" VerticalAlignment="Center" MinWidth="120" Margin="10 0 10 0" />
-      </DockPanel>
-
-      <DockPanel>
-        <TextBlock Text="Output Wallet" />
-        <PathIcon Data="{StaticResource warning_regular}" Foreground="{DynamicResource WarningMessageForeground}"  Margin="-10,0,10,0" Height="20">
+    <StackPanel>
+      <StackPanel Orientation="Horizontal" Spacing="10">
+        <TextBlock Text="Destination Wallet" />
```

Comment:
This would make the setting clear it is about the `Coinjoin To Another Wallet` feature:

```suggestion
        <TextBlock Text="Coinjoin to wallet:" />
```

Or

```suggestion
        <TextBlock Text="Coinjoin to this wallet:" />
```

Reply:
`Coinjoin to this wallet:` sounds better, can you also update the Triangle's tooltip?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/13067#discussion_r1611324497